src/configurations/config.json

```diff
@@ -1,50 +1,21 @@
 {
   "service": {
     "downloadUrl": "https://github.com/Microsoft/sqltoolsservice/releases/download/{#version#}/microsoft.sqltools.servicelayer-{#fileName#}",
-    "version": "4.11.0.2",
+    "version": "5.0.0.4",
     "downloadFileNames": {
-      "Windows_86": "win-x86-net7.0.zip",
-      "Windows_64": "win-x64-net7.0.zip",
-      "Windows_ARM64": "win-arm64-net7.0.zip",
-      "OSX_10_11_64": "osx-x64-net7.0.tar.gz",
-      "OSX_ARM64": "osx-arm64-net7.0.tar.gz",
-      "CentOS_7": "rhel-x64-net7.0.tar.gz",
-      "Debian_8": "rhel-x64-net7.0.tar.gz",
-      "Fedora_23": "rhel-x64-net7.0.tar.gz",
-      "OpenSUSE_13_2": "rhel-x64-net7.0.tar.gz",
-      "SLES_12_2": "rhel-x64-net7.0.tar.gz",
-      "RHEL_7": "rhel-x64-net7.0.tar.gz",
-      "Ubuntu_14": "rhel-x64-net7.0.tar.gz",
-      "Ubuntu_16": "rhel-x64-net7.0.tar.gz",
-      "Linux_ARM64": "linux-arm64-net7.0.tar.gz"
+      "Windows_86": "win-x86-net8.0.zip",
+      "Windows_64": "win-x64-net8.0.zip",
+      "Windows_ARM64": "win-arm64-net8.0.zip",
+      "OSX_64": "osx-x64-net8.0.tar.gz",
+      "OSX_ARM64": "osx-arm64-net8.0.tar.gz",
+      "Linux_64": "linux-x64-net8.0.tar.gz",
+      "Linux_ARM64": "linux-arm64-net8.0.tar.gz"
     },
     "installDir": "../sqltoolsservice/{#version#}/{#platform#}",
     "executableFiles": [
       "MicrosoftSqlToolsServiceLayer.exe",
       "MicrosoftSqlToolsServiceLayer",
       "MicrosoftSqlToolsServiceLayer.dll"
     ]
-  },
-  "v1Service": {
```

Comment:
This is still used for downlevel Mac versions so we should keep it as-is for now

https://github.com/Microsoft/vscode-mssql/blob/79da38b0e928ade250a846a204977519e4f120cf/src/languageservice/serviceclient.ts#L233

Reply:
What would be the rule or trade-off when one could remove such support for these old Mac OS versions? Or is in general the goal to offer maximum support to older versions?

Reply:
We've never really set a goal or deprecation timeline, but that's definitely something we should work on! For now though continuing to support all previous versions is something we strive to adhere to.